docs/argo-server-sso.md

```diff
@@ -108,6 +108,18 @@ If no rule matches, we deny the user access.
     
     The precedence must be the lowest of all your service accounts. 
 
+## SSO Login Time
+
+> v2.12 and after
+
+By default, your SSO session will expire after 10 hours. You can change this by adding an expiry value to your [workflow-controller-configmap.yaml](workflow-controller-configmap.yaml) under the SSO heading.
+
+    ```yaml
+    sso:
+      # Expiry defines how long your login is valid for in hours. (optional)
+      expiry: 240h
```

Comment:
This is `sessionExpiry`.

Reply:
Cheers. I did warn you I'm not great with code!